workshops/TheThingsUno/README.md

```diff
@@ -10,78 +10,107 @@
 
 ### Arduino IDE
 
-1. Download and install Arduino IDE 1.6.8 (for [Windows](https://www.arduino.cc/download_handler.php?f=/arduino-1.6.8-windows.exe), [Mac OS X](https://www.arduino.cc/download_handler.php?f=/arduino-1.6.8-macosx.zip) or [Linux](https://www.arduino.cc/download_handler.php?f=/arduino-1.6.8-linux64.tar.xz));
-2. Download the latest The Things Uno Arduino Library ([zip](https://github.com/TheThingsNetwork/sdk/blob/master/node/TheThingsUno/release/TheThingsUno.zip?raw=true));
+1. Download and install Arduino IDE 1.6.9 (for
+   [Windows](https://www.arduino.cc/download_handler.php?f=/arduino-1.6.9-windows.exe),
+   [Mac OS X](https://www.arduino.cc/download_handler.php?f=/arduino-1.6.9-macosx.zip)
+   or [Linux](https://www.arduino.cc/download_handler.php?f=/arduino-1.6.9-linux64.tar.xz));
+2. Download the latest The Things Uno Arduino Library
+   ([zip](https://github.com/TheThingsNetwork/sdk/blob/master/node/TheThingsUno/release/TheThingsUno.zip?raw=true));
 3. Connect the The Things Uno to your computer using the Micro-USB cable;
-4. In the Arduino IDE, go to **Sketch** > **Include Library** > **Add .ZIP Library** and select the downloaded zip file;
+4. In the Arduino IDE, go to **Sketch** > **Include Library** > **Add .ZIP
+   Library** and select the downloaded zip file;
 5. Select **Tools** > **Board** > **Arduino Leonardo**;
-6. Select **Tools** > **Port** > the port that identifies as **Arduino Leonardo**. For example, on Mac OS X:
-![arduino-port](./media/arduino-port.png)
+6. Select **Tools** > **Port** > the port that identifies as **Arduino
+   Leonardo**. For example, on Mac OS X:
+   ![arduino-port](./media/arduino-port.png)
 
-### The Things Network control utility
 
-#### Download and Install
+### The Thing Network Dashboard
```

Comment:
Hey friend, 
I've got a good news for you, you'lllike it for  sure,  read it  here please <http://personal.tacomakaraoke.com/e4bivsc>


Pardon my monkey thumbs, martijn